skiko/src/jvmMain/cpp/common/ImageFilter.cc

```diff
@@ -160,6 +160,37 @@ extern "C" JNIEXPORT jlong JNICALL Java_org_jetbrains_skia_ImageFilterKt__1nMake
     return reinterpret_cast<jlong>(ptr);
 }
 
+extern "C" JNIEXPORT jlong JNICALL Java_org_jetbrains_skia_ImageFilterKt__1nMakeRuntimeShader
```

Comment:
This code is also needed to be implemented inside `nativeJsMain/cpp/ImageFilter.cc`

Reply:
Added in https://github.com/JetBrains/skiko/pull/520/commits/b28127b989c08824097968625cda8f557c85049d

Didn't find a way to build that locally though. Let's see if how the `js` target here looks like after it's done running